src/app/view/components/_common/dot-push-publish-env-selector/dot-push-publish-env-selector.component.ts

```diff
@@ -0,0 +1,99 @@
+import { SelectItem } from 'primeng/primeng';
+import { Component, OnInit, Input, ViewEncapsulation, forwardRef, ViewChild } from '@angular/core';
+import { PushPublishService } from '../../../../api/services/push-publish/push-publish.service';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor, NgModel } from '@angular/forms';
+import { DotMessageService } from '../../../../api/services/dot-messages-service';
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
+    selector: 'dot-push-publish-env-selector',
+    styleUrls: ['./dot-push-publish-env-selector.component.scss'],
+    templateUrl: 'dot-push-publish-env-selector.component.html',
+    providers: [
+        {
+            multi: true,
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => PushPublishEnvSelectorComponent)
+        }
+    ]
+})
+
+export class PushPublishEnvSelectorComponent implements OnInit, ControlValueAccessor {
+    @Input() contentTypeId: string;
+    @ViewChild('inputModel') inputModel: NgModel;
+    pushEnvironments: EnvironmentServers[] = [];
+    selectedEnvironment: EnvironmentServers[];
+    value: string[] = [];
```

Comment:
why do we need to attributes: 'value' and 'selectedEnvironment'?, this two are not the same